runtime/doc/eval.txt

```diff
@@ -6826,19 +6835,14 @@ synstack({lnum}, {col})					*synstack()*
 		valid positions.
 
 system({cmd} [, {input}])				*system()* *E677*
-		Get the output of the shell command {cmd} as a |string|.  {cmd}
+		Get the output of the command {cmd} as a |string|.  {cmd}
```

Comment:
I assumed the idea here was to point towards the docs for jobstart() rather than repeating the same text.